### DISCUSSION TITLE
Release date for version 7.0?

### DISCUSSION THREAD

**Discussion by PascalSchumacher** [logfellow/logstash-logback-encoder#682] [General]
Hi,

thank you very much for developing logstash-logback-encoder!

As far as I can tell the main branch contains a lot of performance and other improvements.

Any idea when version 7.0 will be released?

**Comment by philsttr** [comment]
Very soon.  Probably in the next couple weeks.  We're trying to wrap it up now.

**Reply by PascalSchumacher** [reply]
Great! I am looking forward to it. Thank you very much!

**Reply by philsttr** [reply]
7.0 released

**Reply by PascalSchumacher** [reply]
Thank you very much! 👍 😃

**Reply by brenuart** [reply]
Don’t hesitate to give feedback in case of issues or if you notice improvements in some areas ;-)
